woocommerce/woocommerce-android
# Call disconnect from a reader when the user clicks cancel during the connection flow

Closes: #7153 


### Description

Call disconnect from a reader when the user clicks cancel during the connection flow. It doesn't stop connection or update right away, but disconnects after the current operation is performed 

### Testing instructions

* Start connection to a reader
* Click on "cancel"
* Notice that it doesn't end up in "connected" state

### Images/gif


https://user-images.githubusercontent.com/4923871/205916211-9ac6ae75-e6d4-43f7-b512-046e80d1f734.mp4
